Kickstarter-iOS/Views/Controllers/PaymentMethodsViewController.swift

```diff
@@ -23,24 +23,21 @@ internal final class PaymentMethodsViewController: UIViewController, MessageBann
     self.tableView.dataSource = self.dataSource
     self.tableView.delegate = self
     self.tableView.register(nib: .CreditCardCell)
-    self.tableView.registerHeaderFooter(nib: .PaymentMethodsFooterView)
-    self.tableView.registerHeaderFooter(nib: .SettingsTableViewHeader)
 
     self.configureHeaderFooterViews()
 
-    self.viewModel.inputs.viewDidLoad()
```

Comment:
Looks like there were duplicates so I've remove those and moved it to the end of this method